Bottom: c01bc37c244cc50f2ab73e0e90d7b923ebcc932c
Top:    b2b126cf6ece972cf4f4fdbdd615dc013cc01daa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-06 00:45:57 +0100

Refresh of hid-gtk-connect-up-the-file-ch

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 1ec824c..5bb8049 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -339,7 +339,7 @@ int ghid_drc_window_throw_dialog (void);
 /* In gui-top-window.c  */
 void ghid_update_toggle_flags (void);
 void ghid_notify_save_pcb (const char *file, bool done);
-void ghid_noiify_pcb_filename_changed (const char *old_filename, const char *new_filename);
+void ghid_noiify_filename_changed (void);
 void ghid_install_accel_groups (GtkWindow *window, GhidGui *gui);
 void ghid_remove_accel_groups (GtkWindow *window, GhidGui *gui);
